db/post_migrate/20240307180905_migrate_devise_two_factor_secrets.rb

```diff
@@ -36,4 +42,22 @@ def down
   def users_with_otp_enabled
     MigrationUser.where(otp_required_for_login: true, otp_secret: nil)
   end
+
+  def abort_with_decryption_error(user)
+    abort <<~MESSAGE
+
+      ERROR: Unable to decrypt OTP secret for user @#{user.account.username}.
```

Comment:
In this spot, isn't `user` an instance of `MigrationUser`, and if so, does `#account` exist?

Reply:
Oof, you're right! Good catch!